db/migrate/20200930185510_add_repository_sources_to_versions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddRepositorySourcesToVersions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :versions, :repository_sources, :jsonb
```

Comment:
Do we really want this to be json or can we simplify it to be an array of strings?

Reply:
I’ve sort of whole hog always used jsonb for arrays since introduction. What do you see as the drawback?